### PR TITLE
add nn-Graph bert

### PR DESCRIPTION
**Pull request by L1aoXingyu**
支持 nn-Graph bert

- [x] 跑通 train&eval in nn-Graph version
- [x] 对齐 eager mode 和 nn-Graph 误差
- [x] 完成 ofrecord dataset
- [x] Graph 中加入 dataset Module
- [x] 加入 lr scheduler
- [x] 完成 README
- [x] 完成 inference.py 代码

### CODE REVIEW

**Comment by L1aoXingyu** [comment]
初步跑了 1000 step 训练，400 step 测试结果如下

losses
trainAcc
trainStepTime
valAcc
valStepTime

**Comment by L1aoXingyu** [comment]
在 Graph 中加入了 OfRecord，跑通训练和测试，还未验证精度

**Comment by L1aoXingyu** [comment]
在训练过程中按 ctrl-c 退出时会出现 core dump，异常退出，生成 core.* 文件

**Comment by L1aoXingyu** [comment]
支持 OfRecord 数据读入，训练 20 epochs 对齐结果如下
losses
trainAcc
trainStepTime
valAcc
valStepTime

初步对齐 graph 和 eager 的结果，不过还未进行正确性验证，因为 eager 的代码是基于非官方的 pytorch 版本进行复现的，未和 lazy 版本以及官方的 tf 版本进行对齐

**Comment by L1aoXingyu** [comment]
`NLLLoss(ignore_index=0)` 在 graph 中不支持，正在尝试解决

**Comment by L1aoXingyu** [comment]
* 和 @CPFLAME 一起解决了 `NLLLoss(ignore_index=0)` 的问题
* 发现 eager 实现下 `next_sent_loss` 计算存在问题，也使用了 ignore_index，而 next_sent_label 输出只有 [0, 1]，所以不应该和 `mask_loss` 一样使用 ignore_index


losses
trainAcc
valAcc

**Comment by L1aoXingyu** [comment]
这个 PR 暂时废弃了，后续的工作在 https://github.com/Oneflow-Inc/models/pull/117 继续开展
